### PR TITLE
lesspipe: Add caveats

### PR DESCRIPTION
**Pull request by hazcod**
Add installation caveat so users know how to start using this package.

### CODE REVIEW

File: Formula/lesspipe.rb [hunk 1]
```diff
@@ -33,4 +33,12 @@ def install
     assert File.exist?("homebrew.tar.gz")
     assert_match /file2.txt/, shell_output("tar tvzf homebrew.tar.gz | #{bin}/tarcolor")
   end
+  
+  def caveats
+    <<-EOS
+      lesspipe.sh gets symlinked to /usr/local/bin/lesspipe.sh.
+      Append the following to your .bash_profile/.bashrc:
+      export LESSOPEN="|/usr/local/bin/lesspipe.sh %s" LESS_ADVANCED_PREPROCESSOR=1
```
Comment (fxcoudert) [inline Formula/lesspipe.rb]: - The first line is not really useful
- Are the instructions Homebrew-specific?
- Please do not hardcode `/usr/local/bin` but use `HOMEBREW_PREFIX` in caveats

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/lesspipe.rb (hunk 1 above)
Comment (fxcoudert) [inline Formula/lesspipe.rb]: `  * C: 36: col 1: Trailing whitespace detected.`

File: Formula/lesspipe.rb [hunk 2]
```diff
@@ -33,4 +33,11 @@ def install
     assert File.exist?("homebrew.tar.gz")
     assert_match /file2.txt/, shell_output("tar tvzf homebrew.tar.gz | #{bin}/tarcolor")
   end
+
+  def caveats
+    <<-EOS
+      Append the following to your .bash_profile/.bashrc:
+      export LESSOPEN="|$(brew --prefix)/bin/lesspipe.sh %s" LESS_ADVANCED_PREPROCESSOR=1
```
Comment (fxcoudert) [inline Formula/lesspipe.rb]: Our current practice for caveats is to use `#{HOMEBREW_PREFIX}` and not `$(brew --prefix)`

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/lesspipe.rb [hunk 3]
```diff
@@ -33,4 +33,11 @@ def install
     assert File.exist?("homebrew.tar.gz")
     assert_match /file2.txt/, shell_output("tar tvzf homebrew.tar.gz | #{bin}/tarcolor")
   end
+
+  def caveats
+    <<-EOS
+      Append the following to your .bash_profile/.bashrc:
```
Comment (ilovezfs) [inline Formula/lesspipe.rb]: This should use `#{shell_profile}`

File: Formula/lesspipe.rb (hunk 2 above)
Comment (HazCod) [inline Formula/lesspipe.rb]: How so? $HOMEBREW_PREFIX is not a global variable.

Comment (fxcoudert) [inline Formula/lesspipe.rb]: As an example look at the caveats for `zsh-navigation-tools.rb`

**Review by fxcoudert — APPROVED** [review]

**Comment by fxcoudert** [comment]
Thanks @HazCod for your contribution to Homebrew! We look forward to seeing you here again :)

**Comment by HazCod** [comment]
It was a tough one, but we'll get there!

**Pull request merged**